runtimes/nodejs/nodejs.md

```diff
@@ -228,12 +205,10 @@ Clever Cloud can inject environment variables that are defined in the
 dashboard and by add-ons linked to your application.
 
 To access your variable in your application, nothing simpler! Just get
-it from your environment, like you would with `PATH`:
-`process.env["MY_VARIABLE"]`.
+it from your environment: `process.env.MY_VARIABLE`.
 
-You can, for example, inject the NODE_ENV variable to define it at "production". It is not
-currently set on our machines, because we need to build the project first. Beware,
-devDependencies do not get fetched if NODE_ENV=production.
+You can, for example, inject the `NODE_ENV` variable to define it at "production". It is not
+currently set on our machines, because we need to build the project first.
```

Comment:
You can (and should) define `NODE_ENV=production`, as many frameworks rely on it for production mode.

«It is not currently set on our machines, because we need to build the project first.» can go IMO, as we force the env with flags during build (IIRC, please check)